Lib/_collections_abc.py

```diff
@@ -409,6 +418,94 @@ def __subclasshook__(cls, C):
         return NotImplemented
 
 
+class _PosArgsGenericAlias(GenericAlias):
+    """ Internal class specifically to represent positional arguments in
+    ``_CallableGenericAlias``.
+    """
+    def __repr__(self):
+        return f"{__name__}._PosArgsGenericAlias" \
+               f"[{', '.join(_type_repr(t) for t in self.__args__)}]"
```

Comment:
```suggestion
        arg = ', '.join(_type_repr(t) for t in self.__args__)
        return f"{__name__}._PosArgsGenericAlias[{arg}]"
```

However, I wonder if it shouldn't just return `f"PosArgs[{arg}]"`, since that is how you would create it.